site/src/components/SettingsHeader/SettingsHeader.tsx

```diff
@@ -18,35 +19,40 @@ export const SettingsHeader: FC<HeaderProps> = ({
 	docsHref,
 	secondary,
 	tooltip,
+	badges,
 }) => {
 	const theme = useTheme();
 
 	return (
 		<Stack alignItems="baseline" direction="row" justifyContent="space-between">
 			<div css={{ maxWidth: 420, marginBottom: 24 }}>
-				<Stack direction="row" spacing={1} alignItems="center">
-					<h1
-						css={[
-							{
-								fontSize: 32,
-								fontWeight: 700,
-								display: "flex",
-								alignItems: "center",
-								lineHeight: "initial",
-								margin: 0,
-								marginBottom: 4,
-								gap: 8,
-							},
-							secondary && {
-								fontSize: 24,
-								fontWeight: 500,
-							},
-						]}
-					>
-						{title}
-					</h1>
-					{tooltip}
+				<Stack direction="row" spacing={2} alignItems="center">
```

Comment:
Only change is that the previous content is in a new stack so that the badges can be placed inside them